### PR TITLE
updated installation guide

### PR DESCRIPTION
**Pull request by zelzhan**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [X] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that another pull request for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [X] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

---

Update the troubleshooting part, regarding the possible errors on libvirt package.

### CODE REVIEW

File: install/installation.md [hunk 1]
```diff
@@ -83,6 +83,13 @@ sudo python bootstrap_gunicorn.py
 * If `gunicorn` boots up successfully, most relevant logs will be stored in
  the `logs` directory. Otherwise they'll likely be in `syslog`.
 
+* If it shows the error regarding the `libvirt`, then there is missing `fftw3.h` file. Try the following:
+```
+sudo apt-get install libvirt-dev
```
Comment (thealphadollar) [inline install/installation.md:88]: Can you give an instance when this error occurs? A screenshot would do.

Comment (zelzhan) [reply install/installation.md:88]: The problem with missing ```fftw3.h``` file occurs on Ubuntu like systems. Have no idea, why this is file is missing out of the box. On windows and CentOS this message does not appear.

About ```sudo apt-get install libvirt-dev```, most of the machines, doesn't have libvirt by default, so I was thinking that this is an essential step which is missed in the installation.

Comment (zelzhan) [reply install/installation.md:88]: This error appears on installation step

Comment (thealphadollar) [reply install/installation.md:88]: Sounds good to me. I asked for the details so that if we ever encounter the same, we know where to start.

Comment (canihavesomecoffee) [reply install/installation.md:88]: Libvirt-bin is installed through the install.sh however, which should (in theory) be enough in combination with the python-libvirt package. I don't mind having the additional note in there though.

**Pull request merged**